build.assets/version-ops/cmd/check/main.go

```diff
@@ -24,15 +24,13 @@ import (
 	"context"
 	"flag"
 	"log"
-	"sort"
 	"strings"
 	"time"
 
 	"golang.org/x/mod/semver"
 
+	"github.com/gravitational/teleport/build.assets/version-ops/lib"
```

Comment:
`lib` is kind of an awkward name for a package. Doesn't tell me anything about what it is or where it comes from. Can we think of a more suitable name?

Reply:
Changed to `lib/github` in cb0ef079ea9c9e18873a662312f96b2a133126e0.